arangod/Pregel/Graph.h

```diff
@@ -41,8 +41,8 @@ struct PregelID {
   std::string key;    // std::string 24
   PregelShard shard;  // uint16_t
 
-  PregelID() : key(""), shard(InvalidPregelShard) {}
-  PregelID(PregelShard s, std::string const& k) : key(k), shard(s) {}
+  PregelID() : shard(InvalidPregelShard) {}
+  PregelID(PregelShard s, std::string k) : key(std::move(k)), shard(s) {}
```

Comment:
This looks dubious to me, did clang-tidy suggest that?

Reply:
Yes. `key("")` because a string is initialized to the empty string by default and `std::move(k)` because the string is anyway copied into the parameter, now it can be reused.